src/AlgebraicGeometry/Curves/MaximalMumfordCurve.jl

```diff
@@ -0,0 +1,238 @@
+################################################################################
+#
+# OSCAR late night complaints list
+# - edges should know if they are directed or undirected
+# - if undirected, Edge(1,3) should be the same as Edge(3,1)
+# - groebner basis of the zero ideal should not be empty (?)
+#
+################################################################################
+
+
+################################################################################
+#
+# Graph curves
+#
+################################################################################
+"""
+TODO: Move check functions to src/Combinatorics/Graphs/functions.jl file
+"""
+function check_i_valency(G::Graph, i::Int)
+  return all(v -> degree(G, v) == i, vertices(G))
+end
+
+function check_i_connectivity(G::Graph, i::Int)
```

Comment:
I think we usually name such function `is_*`, so in this case maybe `is_i_valent` and `is_i_connected`. `check_*` functions usually throw an error if their desired property is not satisfied, e.g. `check_parent(a, b)` checks that `a` and `b` have the same parent and throws otherwise